### PR TITLE
Lenient for BDD added

### PR DESCRIPTION
**Pull request by konradkluz**
Adds lenient for BDDMockito mentioned in issue #1597

### CODE REVIEW

**Comment by konradkluz** [comment]
Hi. @TimvdLippe I have just rebased to the newes release/2.x

File: src/main/java/org/mockito/BDDMockito.java [hunk 1]
```diff
@@ -344,60 +348,68 @@ public void shouldHaveNoMoreInteractions() {
...
          */
-        BDDStubber willCallRealMethod();
+        BDDStrubber willCallRealMethod();
+    }
+
+    /**
+     * see original {@link Stubber}
+     * @since 1.8.0
+     */
+    public interface BDDStrubber extends BaseBDDStubber {
```
Comment (TimvdLippe) [inline src/main/java/org/mockito/BDDMockito.java]: Please rename this back to `BDDStubber` and fix all references to the name.

**Comment by TimvdLippe** [comment]
Closing this as stale. Please address the feedback to reopen the PR.

**Comment by McKratt** [comment]
Still waiting for this PR to be merge. Wihtout it I have to mix notation in the code between BDD notation and none BDD notation.
@TimvdLippe Could you please re open it ?

**Pull request closed without merging**